flipperdevices/flipperzero-firmware
# Upside down / left handed orientation support

# What's new

- rtc flag for orienatation
- gui code making use of the flag

# Verification 

- In system settings set the Lefty orienatation
- verify screen is upside down
- verify directional button actions are inverted to match screen
- verify directional keys work correctly on screen with vertical orientation

# Checklist (For Reviewer)
